recipe/meta.yaml

```diff
@@ -1,45 +1,62 @@
-{% set name = "<PLEASE ADD PKG NAME>" %}
-{% set version = "<PLEASE ADD PKG VERSION>" %}
+{% set name = "thop" %}
+{% set version = "0.1.1.post2209072238" %}
+{% set commit_id = "43c064afb71383501e41eaef9e8c8407265cf77f" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: <sha256>
+  - url: https://pypi.io/packages/py3/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}-py3-none-any.whl
```

Comment:
Could you please drop `whl` and build the package completely from the github `commit_id`?

Reply:
Done, thanks!